load_testing/requirements.txt

```diff
@@ -1,3 +1,2 @@
-locustio
-dotenv
-os
\ No newline at end of file
+locustio==0.9.0
+python-dotenv==0.8.2
```

Comment:
Always a better practise to fix the package versioning and fixed the dotenv package name